.github/workflows/update-snapshots.yaml

```diff
@@ -0,0 +1,50 @@
+name: Update Snapshots
+ 
+on: workflow_dispatch
+
+permissions:
+  contents: write 
+  pull-requests: write 
+
+env:
+  GH_TOKEN: ${{ github.token }}
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: recursive
+    - name: Setup yarn
+      run: |
+        cd ./integration-tests/chopsticks
+        curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
+        export NVM_DIR="$HOME/.nvm" && . "$NVM_DIR/nvm.sh" --no-use
+        eval "[ -f .nvmrc ] && nvm install" && nvm use
+    - name: Install dependencies
+      run: |
+        cd ./integration-tests/chopsticks
+        yarn --immutable
+    - name: Update Snapshots
+      run: |
+        cd ./integration-tests/chopsticks
+        yarn test:CI -u
+    - name: Commit and Create PR
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const branchName = `update-snapshots-${context.sha.slice(0, 7)}`
+          await exec.exec(`git config user.email "devs@kilt.io"`)
+          await exec.exec(`git config user.name "kiltbot"`)
+          await exec.exec(`git checkout -b ${branchName}`)
+          await exec.exec(`git`, ['commit', '-am', 'ci: Update snapshots'])
```

Comment:
If we take any actions based on https://github.com/KILTprotocol/kilt-node/pull/639#discussion_r1604375370, then this should be updated to reflect that.

Reply:
I added the label on hold until we found a solution/decision for the other PR